main.py

```diff
@@ -0,0 +1,34 @@
+from time import sleep
+from sys import platform
+from random import randint
+import signal
+
+from client.comm import Comm
+from module.cli_controller import CLIController
+
+should_stop = False
+
+
+def main():
+    module = CLIController(Comm(), ["modules/swarm_ui/module/commands.json"])
+
+    while not should_stop and not module.stopped:
+        module.process()
+        sleep(0.05)
+
+    module.stop()
+
+
+def stop(signal, frame):
```

Comment:
It seems that signal and frame are unused variables in this function, is this intended?

Reply:
this was taken directly from the readme from python build, they didnt use these variables either. I assume these are required otherwise the function call from the Comm bus would not work